testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -412,6 +412,107 @@ QUnit.test('From renders the right types of editors according to stylingMode opt
     assert.ok($testContainer.find('.dx-field-item .dx-textbox').hasClass('dx-editor-underlined'), 'right class rendered');
 });
 
+QUnit.module('Checkbox editor value', () => {
+    QUnit.test('FormData = {}', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'boolValue', editorType: 'dxCheckBox' } ],
+            formData: { }
+        }).dxForm('instance');
+
+        assert.equal(form.getEditor('boolValue').option('value'), false);
+    });
+
+    QUnit.test('FormData = { innerObject = {} }', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'innerObject.boolValue', editorType: 'dxCheckBox' } ],
+            formData: { innerObject: { } }
+        }).dxForm('instance');
+
+        assert.equal(form.getEditor('innerObject.boolValue').option('value'), false);
+    });
+
+    QUnit.test('FormData = { boolValue: true } -> option("formData", { })', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'boolValue', editorType: 'dxCheckBox' } ],
+            formData: { boolValue: true }
+        }).dxForm('instance');
+
+        form.option('formData', { });
+        assert.equal(form.getEditor('boolValue').option('value'), false);
+    });
+
+    QUnit.test('FormData = { innerObject = { boolValue: true } } -> option("formData", { innerObject = { } })', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'innerObject.boolValue', editorType: 'dxCheckBox' } ],
+            formData: { innerObject: { boolValue: true } }
+        }).dxForm('instance');
+
+        form.option('formData', { innerObject: { } });
+        assert.equal(form.getEditor('innerObject.boolValue').option('value'), false);
+    });
+
+    [true, false, undefined].forEach(boolValue => {
```

Comment:
I think there should be 'no member' entry: [true, false, undefined, *'no member'*]
A CheckBox image for this config should be the same as for the 'undefined' config.

And, we should sync the initial image with the resulting CheckBox image for these operations:
-  'form.option('formData', { });' 
- 'form.option('formData', { boolValue: undefined });' 
- 'form.option('formData', { boolValue: null });'

Reply:
Null value works the same as before.